includes/specials/SpecialUndelete.php

```diff
@@ -1086,6 +1087,7 @@ private function showFile( $key ) {
 	}
 
 	private function showHistory() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "showHistory" has 157 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS138)